### PR TITLE
added a new azure ipi job for control plane workloads

### PR DESCRIPTION
**Pull request by mukrishn**
# changes

- Added a new control-plane-24node job using OOTB Azure IPI install
- Turned off a large scale periodic job due to Azure cloud quota issue with `azure-qe` profile - [failures](https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes/1811771025721921536)

### CODE REVIEW

**Comment by mukrishn** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-24nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: control-plane-ipsec-120nodes
-  cron: 0 4 11,25 * *
+- always_run: false
+  as: control-plane-ipsec-120nodes
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml:26]: Is there a reason here you're turning off this workload? I didnt add this workload but from the title of the PR I would think that you wouldnt edit other items

Comment (mukrishn) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml:26]: we have been getting cloud quota issues with `azure-qe` profile lately, @svetsa-rh suggested to have a new perfscale profile for such large scale test so meanwhile we decided to turn this off - [thread](https://redhat-internal.slack.com/archives/C05CDC19ZKJ/p1720807013727689?thread_ts=1720715581.463399&cid=C05CDC19ZKJ) 
Let me know if you suggest to re-enable it.

failed [jobs](https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes/1811771025721921536)

Comment (mukrishn) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml:26]: I am fine to make it in a separate PR if you prefer that approach.

Comment (paigerube14) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml:26]: no problem, just wasnt clear why this was turning things off

Comment (mukrishn) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml:26]: Updated description, thanks.

**Pull request merged**